pkg/create/templates/test.go

```diff
@@ -43,7 +43,19 @@ container_exists() {
 }
 
 container_ip() {
-  docker inspect --format="{{"{{"}} .NetworkSettings.IPAddress {{"}}"}}" $(cat $cid_file)
+  if (echo "$OSTYPE" | grep -qs 'darwin'); then
```

Comment:
Is it right to assume that everyone on macOS will have the right IP from `docker-machine ip`?

If that's the case, again more Bash nitting, we don't even need `echo` and `grep`:

```bash
case "$OSTYPE" in
  darwin*)  docker-machine ip ;;
  *)        docker inspect --format="{{"{{"}} .NetworkSettings.IPAddress {{"}}"}}" $(cat $cid_file) ;;
esac
```

(inspired by [StackOverflow](http://stackoverflow.com/questions/394230/detect-the-os-from-a-bash-script#394235))